components/scream/src/physics/share/tests/physics_universal_unit_tests.cpp

```diff
@@ -100,14 +100,14 @@ struct UnitWrap::UnitTest<D>::TestUniversal
     const Spack th_temp = physics::T_to_th(T,exner,Smask(true));
     const Spack T_new   = physics::th_to_T(th_temp,exner,Smask(true));
     if (std::abs(T_new[0]-T_in)>tol) {
-      printf("T to th test: abs[ T_new (%.3e) - T_in (%.3e) ]=%e is larger than the tol=%e\n",T_new,T_in,std::abs(T_new[0]-T_in),tol);
+      printf("T to th test: abs[ T_new (%.3e) - T_in (%.3e) ]=%e is larger than the tol=%e\n",T_new[0],T_in,std::abs(T_new[0]-T_in),tol);
       errors++;
     }
     // and vice versa
     const Spack T_temp = physics::th_to_T(T,exner,Smask(true));
     const Spack th_new = physics::T_to_th(T_temp,exner,Smask(true));
-    if (std::abs(T_new[0]-T_in)>tol) {
-      printf("T to th test: abs[ th_new (%.3e) - T_in (%.3e) ]=%e is larger than the tol=%e\n",th_new,T_in,std::abs(th_new[0]-T_in),tol);
+    if (std::abs(th_new[0]-T_in)>tol) {
```

Comment:
Based on the text of the printf, I think a copy paste error was made here, so I fixed it.